windows_containers/windows-containers-release-notes-5-x.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="windows-containers-release-notes-5-x"]
+= Windows Container Support for Red Hat OpenShift release notes
+include::_attributes/common-attributes.adoc[]
+:context: windows-containers-release-notes
+
+toc::[]
+
+[id="about-windows-containers"]
+== About Windows Container Support for Red Hat OpenShift
+
+Windows Container Support for Red Hat OpenShift enables running Windows compute nodes in an {product-title} cluster. Running Windows workloads is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
+
+The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+[id="getting-support"]
+== Getting support
+
+You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
+
+[id="wmco-5-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 5.0.0
+
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 5.0.0 were released in link:https://access.redhat.com/errata/RHBA-2021:3702[RHBA-2021:3702].
```

Comment:
I guess this link needs to point at the WMCO 5.0.0 errata once it will be published, right? Right now it seems to be pointing at the 4.0.0 version.

Reply:
Not sure if this could be the right errata link: https://access.redhat.com/errata/RHSA-2022:0577 (I'm still getting familiar with its usage, I didn't have to play with it in the past)